scripts/clean_osm_data.py

```diff
@@ -168,6 +168,10 @@ def split_cells(df, lst_col="voltage"):
     lst_col : str
         Target column over which to perform the analysis
     """
+
```

Comment:
Let's follow the PEP 257 convention: https://peps.python.org/pep-0257/#multi-line-docstrings
This means you can remove the blank line

Reply:
that was needed for the pre-commit; otherwise it adds it

Reply:
Done, may be worth adding a precommit  rule for that